decidim-admin/app/commands/decidim/admin/destroy_participatory_space_private_user.rb

```diff
@@ -13,6 +13,41 @@ def extra_params
           }
         }
       end
+
+      def run_after_hooks
+        # When private user is destroyed, a hook to destroy the follows of user on private non transparent assembly
+        # or private participatory process and the follows of their children
+        space = resource.privatable_to_type.constantize.find(resource.privatable_to_id)
+
+        return unless space.private_space?
+
+        return if space.respond_to?("is_transparent") && space.is_transparent?
+
+        ids = []
+        follows = Decidim::Follow.where(user: resource.decidim_user_id)
+        ids << find_space_follow_id(follows, resource, space)
+        ids << find_children_follows_ids(follows, space)
+        follows.where(id: ids.flatten.compact).destroy_all if ids.present?
+      end
+
+      def find_space_follow_id(follows, resource, space)
+        follows.where(decidim_followable_type: resource.privatable_to_type)
+               .where(decidim_followable_id: space.id)
+               &.first&.id
+      end
```

Comment:
let's just destroy the object, and avoid carrying around a list of ids.
```suggestion
      def destroy_space_follow(follows, resource, space)
        follows.where(decidim_followable_type: resource.privatable_to_type)
               .where(decidim_followable_id: space.id).destroy_all
      end
```

Reply:
done in a job